### ISSUE TITLE
Running a Python Worker Silently "In The Background" While A Page Loads

### ISSUE THREAD

**Issue by erickbytes** [cloudflare/cloudflare-docs#14533]
### Existing documentation URL(s)

https://developers.cloudflare.com/workers/languages/python/examples/
https://developers.cloudflare.com/workers/examples/

### What changes are you suggesting?

Explain how or demonstrate with an example how to trigger a Python Worker "in the background" with an AJAX script or concurrency so that it doesn't disrupt the normal page loading flow when attached to a route? 

### Additional information

I want to insert to the key value store via a worker simultaneously as the page loads. But when I attach the worker to a route with a routing rule, the response object is rendered to the browser. Working on a Python worker like this but running into too many redirect loops with fetch or redirects:

```
from js import Response, fetch
from datetime import datetime
import os

async def on_fetch(request, env):
    await env.KV_VALUE.put("stuff", str(datetime.now()))
    return await fetch(request.url)
```

**Comment by kodster28** [comment]
Howdy @erickbytes, think the new example in #14675 about `Modifying a Response` might help you here (visible in the [preview](https://docs-py-examples3.cloudflare-docs-7ou.pages.dev/workers/examples/modify-response/)).

TL;DR, when you have an incoming request, you need to create a copy of the request and then modify whatever properties make sense to change.
